node/tests/ServerModules.test.ts

```diff
@@ -280,3 +282,191 @@ describe("GlideJson", () => {
         },
     );
 });
+
+describe("GlideFt", () => {
+    const testsFailed = 0;
+    let cluster: ValkeyCluster;
+    let client: GlideClusterClient;
+    beforeAll(async () => {
+        const clusterAddresses = parseCommandLineArgs()["cluster-endpoints"];
+        cluster = await ValkeyCluster.initFromExistingCluster(
+            true,
+            parseEndpoints(clusterAddresses),
+            getServerVersion,
+        );
+    }, 20000);
+
+    afterEach(async () => {
+        await flushAndCloseClient(true, cluster.getAddresses(), client);
+    });
+
+    afterAll(async () => {
+        if (testsFailed === 0) {
+            await cluster.close();
+        }
+    }, TIMEOUT);
+
+    it("ServerModules check Vector Search module is loaded", async () => {
+        client = await GlideClusterClient.createClient(
+            getClientConfigurationOption(
+                cluster.getAddresses(),
+                ProtocolVersion.RESP3,
+            ),
+        );
+        const info = await client.info({
+            sections: [InfoOptions.Modules],
+            route: "randomNode",
+        });
+        expect(info).toContain("# search_index_stats");
+    });
+
+    it("Ft.Create test", async () => {
```

Comment:
consider deleting all created indices or `flushall` at the end in `finally` block

Reply:
after each test, we flush and close the client using `flushAndCloseClient`.